.github/workflows/ffigen.yml

```diff
@@ -82,8 +86,12 @@ jobs:
       - uses: dart-lang/setup-dart@fedb1266e91cf51be2fdb382869461a434b920a3
```

Comment:
You can remove this now, since it's going to be included in the `flutter-action`.